Traceback (most recent call last):
  File "/usr/local/python/3.12.1/lib/python3.12/site-packages/jupyter_cache/executors/utils.py", line 58, in single_nb_execution
    executenb(
  File "/home/codespace/.local/lib/python3.12/site-packages/nbclient/client.py", line 1319, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/codespace/.local/lib/python3.12/site-packages/jupyter_core/utils/__init__.py", line 165, in wrapped
    return loop.run_until_complete(inner)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/python/3.12.1/lib/python3.12/asyncio/base_events.py", line 684, in run_until_complete
    return future.result()
           ^^^^^^^^^^^^^^^
  File "/home/codespace/.local/lib/python3.12/site-packages/nbclient/client.py", line 709, in async_execute
    await self.async_execute_cell(
  File "/home/codespace/.local/lib/python3.12/site-packages/nbclient/client.py", line 1062, in async_execute_cell
    await self._check_raise_for_error(cell, cell_index, exec_reply)
  File "/home/codespace/.local/lib/python3.12/site-packages/nbclient/client.py", line 918, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
# Koneksi ke Aiven MySQL
mysql_engine = create_engine("mysql+pymysql://avnadmin:AVNS_avYU0WPbd4YEsvg2t9b@iris-mysql-iris-sql.l.aivencloud.com:19560/defaultdb")
# Koneksi ke Aiven PostgreSQL
postgres_engine = create_engine("postgresql+psycopg2://avnadmin:AVNS_qWgY6go04QEsv9VSVjz@pg-fc79532-iris-postgre.g.aivencloud.com:18594/defaultdb")

# Mengambil data sari MySQL
df_mysql = pd.read_sql("SELECT * FROM iris_sql",mysql_engine)
df_mysql.head()

------------------


[31m---------------------------------------------------------------------------[39m
[31mProgrammingError[39m                          Traceback (most recent call last)
[36mFile [39m[32m/usr/local/python/3.12.1/lib/python3.12/site-packages/sqlalchemy/engine/base.py:1963[39m, in [36mConnection._exec_single_context[39m[34m(self, dialect, context, statement, parameters)[39m
[32m   1962[39m     [38;5;28;01mif[39;00m [38;5;129;01mnot[39;00m evt_handled:
[32m-> [39m[32m1963[39m         [38;5;28;43mself[39;49m[43m.[49m[43mdialect[49m[43m.[49m[43mdo_execute[49m[43m([49m
[32m   1964[39m [43m            [49m[43mcursor[49m[43m,[49m[43m [49m[43mstr_statement[49m[43m,[49m[43m [49m[43meffective_parameters[49m[43m,[49m[43m [49m[43mcontext[49m
[32m   1965[39m [43m        [49m[43m)[49m
[32m   1967[39m [38;5;28;01mif[39;00m [38;5;28mself[39m._has_events [38;5;129;01mor[39;00m [38;5;28mself[39m.engine._has_events:

[36mFile [39m[32m/usr/local/python/3.12.1/lib/python3.12/site-packages/sqlalchemy/engine/default.py:943[39m, in [36mDefaultDialect.do_execute[39m[34m(self, cursor, statement, parameters, context)[39m
[32m    942[39m [38;5;28;01mdef[39;00m[38;5;250m [39m[34mdo_execute[39m([38;5;28mself[39m, cursor, statement, parameters, context=[38;5;28;01mNone[39;00m):
[32m--> [39m[32m943[39m     [43mcursor[49m[43m.[49m[43mexecute[49m[43m([49m[43mstatement[49m[43m,[49m[43m [49m[43mparameters[49m[43m)[49m

[36mFile [39m[32m/usr/local/python/3.12.1/lib/python3.12/site-packages/pymysql/cursors.py:153[39m, in [36mCursor.execute[39m[34m(self, query, args)[39m
[32m    151[39m query = [38;5;28mself[39m.mogrify(query, args)
[32m--> [39m[32m153[39m result = [38;5;28;43mself[39;49m[43m.[49m[43m_query[49m[43m([49m[43mquery[49m[43m)[49m
[32m    154[39m [38;5;28mself[39m._executed = query

[36mFile [39m[32m/usr/local/python/3.12.1/lib/python3.12/site-packages/pymysql/cursors.py:322[39m, in [36mCursor._query[39m[34m(self, q)[39m
[32m    321[39m [38;5;28mself[39m._clear_result()
[32m--> [39m[32m322[39m [43mconn[49m[43m.[49m[43mquery[49m[43m([49m[43mq[49m[43m)[49m
[32m    323[39m [38;5;28mself[39m._do_get_result()

[36mFile [39m[32m/usr/local/python/3.12.1/lib/python3.12/site-packages/pymysql/connections.py:563[39m, in [36mConnection.query[39m[34m(self, sql, unbuffered)[39m
[32m    562[39m [38;5;28mself[39m._execute_command(COMMAND.COM_QUERY, sql)
[32m--> [39m[32m563[39m [38;5;28mself[39m._affected_rows = [38;5;28;43mself[39;49m[43m.[49m[43m_read_query_result[49m[43m([49m[43munbuffered[49m[43m=[49m[43munbuffered[49m[43m)[49m
[32m    564[39m [38;5;28;01mreturn[39;00m [38;5;28mself[39m._affected_rows

[36mFile [39m[32m/usr/local/python/3.12.1/lib/python3.12/site-packages/pymysql/connections.py:825[39m, in [36mConnection._read_query_result[39m[34m(self, unbuffered)[39m
[32m    824[39m     result = MySQLResult([38;5;28mself[39m)
[32m--> [39m[32m825[39m     [43mresult[49m[43m.[49m[43mread[49m[43m([49m[43m)[49m
[32m    826[39m [38;5;28mself[39m._result = result

[36mFile [39m[32m/usr/local/python/3.12.1/lib/python3.12/site-packages/pymysql/connections.py:1199[39m, in [36mMySQLResult.read[39m[34m(self)[39m
[32m   1198[39m [38;5;28;01mtry[39;00m:
[32m-> [39m[32m1199[39m     first_packet = [38;5;28;43mself[39;49m[43m.[49m[43mconnection[49m[43m.[49m[43m_read_packet[49m[43m([49m[43m)[49m
[32m   1201[39m     [38;5;28;01mif[39;00m first_packet.is_ok_packet():

[36mFile [39m[32m/usr/local/python/3.12.1/lib/python3.12/site-packages/pymysql/connections.py:775[39m, in [36mConnection._read_packet[39m[34m(self, packet_type)[39m
[32m    774[39m         [38;5;28mself[39m._result.unbuffered_active = [38;5;28;01mFalse[39;00m
[32m--> [39m[32m775[39m     [43mpacket[49m[43m.[49m[43mraise_for_error[49m[43m([49m[43m)[49m
[32m    776[39m [38;5;28;01mreturn[39;00m packet

[36mFile [39m[32m/usr/local/python/3.12.1/lib/python3.12/site-packages/pymysql/protocol.py:219[39m, in [36mMysqlPacket.raise_for_error[39m[34m(self)[39m
[32m    218[39m     [38;5;28mprint[39m([33m"[39m[33merrno =[39m[33m"[39m, errno)
[32m--> [39m[32m219[39m [43merr[49m[43m.[49m[43mraise_mysql_exception[49m[43m([49m[38;5;28;43mself[39;49m[43m.[49m[43m_data[49m[43m)[49m

[36mFile [39m[32m/usr/local/python/3.12.1/lib/python3.12/site-packages/pymysql/err.py:150[39m, in [36mraise_mysql_exception[39m[34m(data)[39m
[32m    149[39m     errorclass = InternalError [38;5;28;01mif[39;00m errno < [32m1000[39m [38;5;28;01melse[39;00m OperationalError
[32m--> [39m[32m150[39m [38;5;28;01mraise[39;00m errorclass(errno, errval)

[31mProgrammingError[39m: (1146, "Table 'defaultdb.iris_sql' doesn't exist")

The above exception was the direct cause of the following exception:

[31mProgrammingError[39m                          Traceback (most recent call last)
[36mCell[39m[36m [39m[32mIn[2][39m[32m, line 7[39m
[32m      4[39m postgres_engine = create_engine([33m"[39m[33mpostgresql+psycopg2://avnadmin:AVNS_qWgY6go04QEsv9VSVjz@pg-fc79532-iris-postgre.g.aivencloud.com:18594/defaultdb[39m[33m"[39m)
[32m      6[39m [38;5;66;03m# Mengambil data sari MySQL[39;00m
[32m----> [39m[32m7[39m df_mysql = [43mpd[49m[43m.[49m[43mread_sql[49m[43m([49m[33;43m"[39;49m[33;43mSELECT * FROM iris_sql[39;49m[33;43m"[39;49m[43m,[49m[43mmysql_engine[49m[43m)[49m
[32m      8[39m df_mysql.head()

[36mFile [39m[32m~/.local/lib/python3.12/site-packages/pandas/io/sql.py:734[39m, in [36mread_sql[39m[34m(sql, con, index_col, coerce_float, params, parse_dates, columns, chunksize, dtype_backend, dtype)[39m
[32m    724[39m     [38;5;28;01mreturn[39;00m pandas_sql.read_table(
[32m    725[39m         sql,
[32m    726[39m         index_col=index_col,
[32m   (...)[39m[32m    731[39m         dtype_backend=dtype_backend,
[32m    732[39m     )
[32m    733[39m [38;5;28;01melse[39;00m:
[32m--> [39m[32m734[39m     [38;5;28;01mreturn[39;00m [43mpandas_sql[49m[43m.[49m[43mread_query[49m[43m([49m
[32m    735[39m [43m        [49m[43msql[49m[43m,[49m
[32m    736[39m [43m        [49m[43mindex_col[49m[43m=[49m[43mindex_col[49m[43m,[49m
[32m    737[39m [43m        [49m[43mparams[49m[43m=[49m[43mparams[49m[43m,[49m
[32m    738[39m [43m        [49m[43mcoerce_float[49m[43m=[49m[43mcoerce_float[49m[43m,[49m
[32m    739[39m [43m        [49m[43mparse_dates[49m[43m=[49m[43mparse_dates[49m[43m,[49m
[32m    740[39m [43m        [49m[43mchunksize[49m[43m=[49m[43mchunksize[49m[43m,[49m
[32m    741[39m [43m        [49m[43mdtype_backend[49m[43m=[49m[43mdtype_backend[49m[43m,[49m
[32m    742[39m [43m        [49m[43mdtype[49m[43m=[49m[43mdtype[49m[43m,[49m
[32m    743[39m [43m    [49m[43m)[49m

[36mFile [39m[32m~/.local/lib/python3.12/site-packages/pandas/io/sql.py:1836[39m, in [36mSQLDatabase.read_query[39m[34m(self, sql, index_col, coerce_float, parse_dates, params, chunksize, dtype, dtype_backend)[39m
[32m   1779[39m [38;5;28;01mdef[39;00m[38;5;250m [39m[34mread_query[39m(
[32m   1780[39m     [38;5;28mself[39m,
[32m   1781[39m     sql: [38;5;28mstr[39m,
[32m   (...)[39m[32m   1788[39m     dtype_backend: DtypeBackend | Literal[[33m"[39m[33mnumpy[39m[33m"[39m] = [33m"[39m[33mnumpy[39m[33m"[39m,
[32m   1789[39m ) -> DataFrame | Iterator[DataFrame]:
[32m   1790[39m [38;5;250m    [39m[33;03m"""[39;00m
[32m   1791[39m [33;03m    Read SQL query into a DataFrame.[39;00m
[32m   1792[39m 
[32m   (...)[39m[32m   1834[39m 
[32m   1835[39m [33;03m    """[39;00m
[32m-> [39m[32m1836[39m     result = [38;5;28;43mself[39;49m[43m.[49m[43mexecute[49m[43m([49m[43msql[49m[43m,[49m[43m [49m[43mparams[49m[43m)[49m
[32m   1837[39m     columns = result.keys()
[32m   1839[39m     [38;5;28;01mif[39;00m chunksize [38;5;129;01mis[39;00m [38;5;129;01mnot[39;00m [38;5;28;01mNone[39;00m:

[36mFile [39m[32m~/.local/lib/python3.12/site-packages/pandas/io/sql.py:1659[39m, in [36mSQLDatabase.execute[39m[34m(self, sql, params)[39m
[32m   1657[39m args = [] [38;5;28;01mif[39;00m params [38;5;129;01mis[39;00m [38;5;28;01mNone[39;00m [38;5;28;01melse[39;00m [params]
[32m   1658[39m [38;5;28;01mif[39;00m [38;5;28misinstance[39m(sql, [38;5;28mstr[39m):
[32m-> [39m[32m1659[39m     [38;5;28;01mreturn[39;00m [38;5;28;43mself[39;49m[43m.[49m[43mcon[49m[43m.[49m[43mexec_driver_sql[49m[43m([49m[43msql[49m[43m,[49m[43m [49m[43m*[49m[43margs[49m[43m)[49m
[32m   1660[39m [38;5;28;01mreturn[39;00m [38;5;28mself[39m.con.execute(sql, *args)

[36mFile [39m[32m/usr/local/python/3.12.1/lib/python3.12/site-packages/sqlalchemy/engine/base.py:1775[39m, in [36mConnection.exec_driver_sql[39m[34m(self, statement, parameters, execution_options)[39m
[32m   1770[39m execution_options = [38;5;28mself[39m._execution_options.merge_with(
[32m   1771[39m     execution_options
[32m   1772[39m )
[32m   1774[39m dialect = [38;5;28mself[39m.dialect
[32m-> [39m[32m1775[39m ret = [38;5;28;43mself[39;49m[43m.[49m[43m_execute_context[49m[43m([49m
[32m   1776[39m [43m    [49m[43mdialect[49m[43m,[49m
[32m   1777[39m [43m    [49m[43mdialect[49m[43m.[49m[43mexecution_ctx_cls[49m[43m.[49m[43m_init_statement[49m[43m,[49m
[32m   1778[39m [43m    [49m[43mstatement[49m[43m,[49m
[32m   1779[39m [43m    [49m[38;5;28;43;01mNone[39;49;00m[43m,[49m
[32m   1780[39m [43m    [49m[43mexecution_options[49m[43m,[49m
[32m   1781[39m [43m    [49m[43mstatement[49m[43m,[49m
[32m   1782[39m [43m    [49m[43mdistilled_parameters[49m[43m,[49m
[32m   1783[39m [43m[49m[43m)[49m
[32m   1785[39m [38;5;28;01mreturn[39;00m ret

[36mFile [39m[32m/usr/local/python/3.12.1/lib/python3.12/site-packages/sqlalchemy/engine/base.py:1842[39m, in [36mConnection._execute_context[39m[34m(self, dialect, constructor, statement, parameters, execution_options, *args, **kw)[39m
[32m   1840[39m     [38;5;28;01mreturn[39;00m [38;5;28mself[39m._exec_insertmany_context(dialect, context)
[32m   1841[39m [38;5;28;01melse[39;00m:
[32m-> [39m[32m1842[39m     [38;5;28;01mreturn[39;00m [38;5;28;43mself[39;49m[43m.[49m[43m_exec_single_context[49m[43m([49m
[32m   1843[39m [43m        [49m[43mdialect[49m[43m,[49m[43m [49m[43mcontext[49m[43m,[49m[43m [49m[43mstatement[49m[43m,[49m[43m [49m[43mparameters[49m
[32m   1844[39m [43m    [49m[43m)[49m

[36mFile [39m[32m/usr/local/python/3.12.1/lib/python3.12/site-packages/sqlalchemy/engine/base.py:1982[39m, in [36mConnection._exec_single_context[39m[34m(self, dialect, context, statement, parameters)[39m
[32m   1979[39m     result = context._setup_result_proxy()
[32m   1981[39m [38;5;28;01mexcept[39;00m [38;5;167;01mBaseException[39;00m [38;5;28;01mas[39;00m e:
[32m-> [39m[32m1982[39m     [38;5;28;43mself[39;49m[43m.[49m[43m_handle_dbapi_exception[49m[43m([49m
[32m   1983[39m [43m        [49m[43me[49m[43m,[49m[43m [49m[43mstr_statement[49m[43m,[49m[43m [49m[43meffective_parameters[49m[43m,[49m[43m [49m[43mcursor[49m[43m,[49m[43m [49m[43mcontext[49m
[32m   1984[39m [43m    [49m[43m)[49m
[32m   1986[39m [38;5;28;01mreturn[39;00m result

[36mFile [39m[32m/usr/local/python/3.12.1/lib/python3.12/site-packages/sqlalchemy/engine/base.py:2351[39m, in [36mConnection._handle_dbapi_exception[39m[34m(self, e, statement, parameters, cursor, context, is_sub_exec)[39m
[32m   2349[39m [38;5;28;01melif[39;00m should_wrap:
[32m   2350[39m     [38;5;28;01massert[39;00m sqlalchemy_exception [38;5;129;01mis[39;00m [38;5;129;01mnot[39;00m [38;5;28;01mNone[39;00m
[32m-> [39m[32m2351[39m     [38;5;28;01mraise[39;00m sqlalchemy_exception.with_traceback(exc_info[[32m2[39m]) [38;5;28;01mfrom[39;00m[38;5;250m [39m[34;01me[39;00m
[32m   2352[39m [38;5;28;01melse[39;00m:
[32m   2353[39m     [38;5;28;01massert[39;00m exc_info[[32m1[39m] [38;5;129;01mis[39;00m [38;5;129;01mnot[39;00m [38;5;28;01mNone[39;00m

[36mFile [39m[32m/usr/local/python/3.12.1/lib/python3.12/site-packages/sqlalchemy/engine/base.py:1963[39m, in [36mConnection._exec_single_context[39m[34m(self, dialect, context, statement, parameters)[39m
[32m   1961[39m                 [38;5;28;01mbreak[39;00m
[32m   1962[39m     [38;5;28;01mif[39;00m [38;5;129;01mnot[39;00m evt_handled:
[32m-> [39m[32m1963[39m         [38;5;28;43mself[39;49m[43m.[49m[43mdialect[49m[43m.[49m[43mdo_execute[49m[43m([49m
[32m   1964[39m [43m            [49m[43mcursor[49m[43m,[49m[43m [49m[43mstr_statement[49m[43m,[49m[43m [49m[43meffective_parameters[49m[43m,[49m[43m [49m[43mcontext[49m
[32m   1965[39m [43m        [49m[43m)[49m
[32m   1967[39m [38;5;28;01mif[39;00m [38;5;28mself[39m._has_events [38;5;129;01mor[39;00m [38;5;28mself[39m.engine._has_events:
[32m   1968[39m     [38;5;28mself[39m.dispatch.after_cursor_execute(
[32m   1969[39m         [38;5;28mself[39m,
[32m   1970[39m         cursor,
[32m   (...)[39m[32m   1974[39m         context.executemany,
[32m   1975[39m     )

[36mFile [39m[32m/usr/local/python/3.12.1/lib/python3.12/site-packages/sqlalchemy/engine/default.py:943[39m, in [36mDefaultDialect.do_execute[39m[34m(self, cursor, statement, parameters, context)[39m
[32m    942[39m [38;5;28;01mdef[39;00m[38;5;250m [39m[34mdo_execute[39m([38;5;28mself[39m, cursor, statement, parameters, context=[38;5;28;01mNone[39;00m):
[32m--> [39m[32m943[39m     [43mcursor[49m[43m.[49m[43mexecute[49m[43m([49m[43mstatement[49m[43m,[49m[43m [49m[43mparameters[49m[43m)[49m

[36mFile [39m[32m/usr/local/python/3.12.1/lib/python3.12/site-packages/pymysql/cursors.py:153[39m, in [36mCursor.execute[39m[34m(self, query, args)[39m
[32m    149[39m     [38;5;28;01mpass[39;00m
[32m    151[39m query = [38;5;28mself[39m.mogrify(query, args)
[32m--> [39m[32m153[39m result = [38;5;28;43mself[39;49m[43m.[49m[43m_query[49m[43m([49m[43mquery[49m[43m)[49m
[32m    154[39m [38;5;28mself[39m._executed = query
[32m    155[39m [38;5;28;01mreturn[39;00m result

[36mFile [39m[32m/usr/local/python/3.12.1/lib/python3.12/site-packages/pymysql/cursors.py:322[39m, in [36mCursor._query[39m[34m(self, q)[39m
[32m    320[39m conn = [38;5;28mself[39m._get_db()
[32m    321[39m [38;5;28mself[39m._clear_result()
[32m--> [39m[32m322[39m [43mconn[49m[43m.[49m[43mquery[49m[43m([49m[43mq[49m[43m)[49m
[32m    323[39m [38;5;28mself[39m._do_get_result()
[32m    324[39m [38;5;28;01mreturn[39;00m [38;5;28mself[39m.rowcount

[36mFile [39m[32m/usr/local/python/3.12.1/lib/python3.12/site-packages/pymysql/connections.py:563[39m, in [36mConnection.query[39m[34m(self, sql, unbuffered)[39m
[32m    561[39m     sql = sql.encode([38;5;28mself[39m.encoding, [33m"[39m[33msurrogateescape[39m[33m"[39m)
[32m    562[39m [38;5;28mself[39m._execute_command(COMMAND.COM_QUERY, sql)
[32m--> [39m[32m563[39m [38;5;28mself[39m._affected_rows = [38;5;28;43mself[39;49m[43m.[49m[43m_read_query_result[49m[43m([49m[43munbuffered[49m[43m=[49m[43munbuffered[49m[43m)[49m
[32m    564[39m [38;5;28;01mreturn[39;00m [38;5;28mself[39m._affected_rows

[36mFile [39m[32m/usr/local/python/3.12.1/lib/python3.12/site-packages/pymysql/connections.py:825[39m, in [36mConnection._read_query_result[39m[34m(self, unbuffered)[39m
[32m    823[39m [38;5;28;01melse[39;00m:
[32m    824[39m     result = MySQLResult([38;5;28mself[39m)
[32m--> [39m[32m825[39m     [43mresult[49m[43m.[49m[43mread[49m[43m([49m[43m)[49m
[32m    826[39m [38;5;28mself[39m._result = result
[32m    827[39m [38;5;28;01mif[39;00m result.server_status [38;5;129;01mis[39;00m [38;5;129;01mnot[39;00m [38;5;28;01mNone[39;00m:

[36mFile [39m[32m/usr/local/python/3.12.1/lib/python3.12/site-packages/pymysql/connections.py:1199[39m, in [36mMySQLResult.read[39m[34m(self)[39m
[32m   1197[39m [38;5;28;01mdef[39;00m[38;5;250m [39m[34mread[39m([38;5;28mself[39m):
[32m   1198[39m     [38;5;28;01mtry[39;00m:
[32m-> [39m[32m1199[39m         first_packet = [38;5;28;43mself[39;49m[43m.[49m[43mconnection[49m[43m.[49m[43m_read_packet[49m[43m([49m[43m)[49m
[32m   1201[39m         [38;5;28;01mif[39;00m first_packet.is_ok_packet():
[32m   1202[39m             [38;5;28mself[39m._read_ok_packet(first_packet)

[36mFile [39m[32m/usr/local/python/3.12.1/lib/python3.12/site-packages/pymysql/connections.py:775[39m, in [36mConnection._read_packet[39m[34m(self, packet_type)[39m
[32m    773[39m     [38;5;28;01mif[39;00m [38;5;28mself[39m._result [38;5;129;01mis[39;00m [38;5;129;01mnot[39;00m [38;5;28;01mNone[39;00m [38;5;129;01mand[39;00m [38;5;28mself[39m._result.unbuffered_active [38;5;129;01mis[39;00m [38;5;28;01mTrue[39;00m:
[32m    774[39m         [38;5;28mself[39m._result.unbuffered_active = [38;5;28;01mFalse[39;00m
[32m--> [39m[32m775[39m     [43mpacket[49m[43m.[49m[43mraise_for_error[49m[43m([49m[43m)[49m
[32m    776[39m [38;5;28;01mreturn[39;00m packet

[36mFile [39m[32m/usr/local/python/3.12.1/lib/python3.12/site-packages/pymysql/protocol.py:219[39m, in [36mMysqlPacket.raise_for_error[39m[34m(self)[39m
[32m    217[39m [38;5;28;01mif[39;00m DEBUG:
[32m    218[39m     [38;5;28mprint[39m([33m"[39m[33merrno =[39m[33m"[39m, errno)
[32m--> [39m[32m219[39m [43merr[49m[43m.[49m[43mraise_mysql_exception[49m[43m([49m[38;5;28;43mself[39;49m[43m.[49m[43m_data[49m[43m)[49m

[36mFile [39m[32m/usr/local/python/3.12.1/lib/python3.12/site-packages/pymysql/err.py:150[39m, in [36mraise_mysql_exception[39m[34m(data)[39m
[32m    148[39m [38;5;28;01mif[39;00m errorclass [38;5;129;01mis[39;00m [38;5;28;01mNone[39;00m:
[32m    149[39m     errorclass = InternalError [38;5;28;01mif[39;00m errno < [32m1000[39m [38;5;28;01melse[39;00m OperationalError
[32m--> [39m[32m150[39m [38;5;28;01mraise[39;00m errorclass(errno, errval)

[31mProgrammingError[39m: (pymysql.err.ProgrammingError) (1146, "Table 'defaultdb.iris_sql' doesn't exist")
[SQL: SELECT * FROM iris_sql]
(Background on this error at: https://sqlalche.me/e/20/f405)

